### ISSUE TITLE
Phpmyadmin uses backticks (`) instead of "

### ISSUE THREAD

**Issue by Nilsbijlsma** [2do2go/json-sql#4]
Is it possible to use json-sql for phpmyadmin, since that service uses backticks and my sql contsructed querys consist of double quotes. Besides: How can I retreive the query with the values instead of $p1 $p2 and so on?

**Comment by artzhuchka** [comment]
Phpmyadmin is a client for MySQL, I added support of MySQL dialect at commit f751569b6b40997e1b78dfba8c09e1f2a4293ca0 and it available now in json-sql v0.2.5.
You can update json-sql to new version from npm and use 'mysql' dialect:

```javascript
var jsonSql = require('./lib')({dialect: 'mysql'});
var sql = jsonSql.build({type: 'select', fields: ['a', 'b'], table: 'test'});
sql.query // select `a`, `b` from `test`;
```

Answer to your second question you can find here: https://github.com/2do2go/json-sql/pull/3

**Comment by Nilsbijlsma** [comment]
Missed the dialect inclusion. Thanks. :)
